pom.xml

```diff
@@ -1457,6 +1525,14 @@
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
how do we ensure this only executes in the modules that use guava?

What happens in the modules that need to shade more than guava?